asv_bench/benchmarks/indexing.py

```diff
@@ -287,3 +287,13 @@ def setup(self):
 
     def time_subset(self):
         self.p.ix[(self.inds, self.inds, self.inds)]
+
+
+class IndexerLookup(object):
+    goal_time = 0.2
+
+    def setup(self):
+        self.s = Series(range(10))
```

Comment:
can you add for ``.loc`` (and might as well for ``.ix``)

Reply:
Ok, done.